### DISCUSSION TITLE
求教关于视频分块上传失败问题

### DISCUSSION THREAD

**Discussion by Naxrin** [Nemo2011/bilibili-api#56] [Show and tell]
写出来能跑但显示“分块上传失败”，分块一块都传不上去，求教一下大佬，可能是什么问题，可以怎么修改？以及还想问一下这里的api有没有投稿编辑功能呢？

**Comment by Nemo2011** [comment]
可能是 aiohttp 的 ServerDisconnected, 找个时间改成 httpx 实现就可以了

**Comment by Nemo2011** [comment]
改好了, `12.4.0`

**Reply by Naxrin** [reply]
我这边还是没法上传，循环输出CHUNK_FAILED（全文在report.txt），代码在code.txt.稍微改了一点不知道有没改错，编译器没报错就是了
[code.txt](https://github.com/Nemo2011/bilibili-api/files/9567069/code.txt)
[report.txt](https://github.com/Nemo2011/bilibili-api/files/9567071/report.txt)

**Comment by Nemo2011** [comment]
meta["videos"] 和 meta["cover"] 保留

**Reply by Naxrin** [reply]
超级感谢！视频上传已经解决了。以及想问一下这个api有没有方法修改稿件呢（改标题简介，改分P名，改分P顺序，增加删除分P，等等的）

**Reply by Nemo2011** [reply]
以后会有的。

**Reply by Naxrin** [reply]
期待哇

**Reply by Naxrin** [reply]
> 以后会有的。

大概多久会有呢，想做个项目要用上改投稿但是初学编程只会语法不懂api爬取（悲）
无意催（抱歉），或者有相关的书籍推荐吗？

**Reply by XUAN-CSU** [reply]
Naxrin你好，请问您最终是如何修改的呢？我也出现了这个问题

**Reply by Naxrin** [reply]
> Naxrin你好，请问您最终是如何修改的呢？我也出现了这个问题

你可以先试试翻git上的Codes按照meta参数示例挨个补上文档示例缺的东西，meta["videos"] 和 meta["cover"] 留空字符串但不能不写

**Reply by XUAN-CSU** [reply]
Naxrin你好，我想问一下，你现在还能上传视频吗？我的是先出现一些PREUPLOAD，然后是code 21150,投稿入口升级中，请重新编辑稿件。

**Reply by XUAN-CSU** [reply]
如果您还能上传，能否贴一下您的上传部分代码呢？谢谢

**Reply by Naxrin** [reply]
> 如果您还能上传，能否贴一下您的上传部分代码呢？谢谢

前天vpn炸了没法回复，我试了几种填法，但也不行了，回到了chunk failed的状态。问nemo作者吧（
